src/main/scala/org/ergoplatform/nodeView/wallet/persistence/WalletRegistry.scala

```diff
@@ -111,7 +124,7 @@ class WalletRegistry(store: LDBVersionedStore)(ws: WalletSettings) extends Score
     * @param heightTo   - max inclusion height of unspent boxes
     * @return sequences of scan-related unspent boxes found in the database
     */
-  def unspentBoxesByInclusionHeight(scanId: ScanId, heightFrom: Height, heightTo: Height): Seq[TrackedBox] =
+  def unspentBoxesByInclusionHeight(scanId: ScanId, heightFrom: Height, heightTo: Height): Array[TrackedBox] =
```

Comment:
All changes from Seq to Array in this file are not motivated. It is better to keep Seq whenever possible, as it is immutable data structure. While at the same time, for performance, the actual representation of Seq can always be Array.